spi/src/main/java/org/opensearch/jobscheduler/spi/schedule/CronSchedule.java

```diff
@@ -96,19 +105,24 @@ String getCronExpression() {
         return this.expression;
     }
 
+    public long getDelay() { return this.scheduleDelay; }
+
+    public void setDelay(long delay) { this.scheduleDelay = delay; }
```

Comment:
curious: why is the set method needed? we don't change the delay once its set during initialization right?

Reply:
When initializing a rollup the delay parameter is included separately from the schedule, so I use this setter to add the delay into the schedule after it is created by the parser. The use of the setter is here: https://github.com/opensearch-project/index-management/pull/147/files#diff-8aaa165b68190cf1eaab9b0c167746357d53f7b23b2ffef9046e5ba51c2cab23
Do you think that it would be better to just create a new schedule with the delay instead of modifying the delay of an existing schedule?